src/pages/data-entry/DataEntry.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import i18n from "../../locales";
+import PageHeader from "../../components/page-header/PageHeader";
+import { useHistory } from "react-router";
+import { History } from "history";
+
+function goTo(history: History, url: string) {
+    history.push(url);
+}
+
+function getConfig() {
+    const help = i18n.t(
```

Comment:
i18n en.pot file was not updated. You can run `yarn localize` to updated the pot file.